docs/macros_schema.md

```diff
@@ -0,0 +1,21 @@
+{% docs ref %}
+    This macro is the default dbt macro  for the model. ref is used in the sql file to refer to the model name.
```

Comment:
This macro overrides the default 'ref' dbt macro when in your local dbt project. This is used in your models to reference other models

Reply:
donw